src/rules/semicolonRule.ts

```diff
@@ -21,6 +21,7 @@ import * as Lint from "../lint";
 
 const OPTION_ALWAYS = "always";
 const OPTION_NEVER = "never";
+const OPTION_ALLOW_BOUND_CLASS_METHODS = "allow-bound-class-methods";
```

Comment:
oh wait, we already have an "ignore-interfaces" option, this should be "ignore-bound-class-methods" then